plugins/pipeline_plugin/transform/general_transformers/cod_transform.py

```diff
@@ -0,0 +1,11 @@
+import geopandas as gpd
+
+
+def transform_cod(input_filename, schema_mapping):
+    df = gpd.read_file(f"zip://{input_filename}")
+
+    # COD data has some NAs
```

Comment:
This comment is unclear. Does this mean that the purpose of the next line (line8) is to filter out any rows where the geometry is null? Is this the desired behaviour?